openwisp_controller/config/base/device.py

```diff
@@ -118,6 +120,11 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        # if WhoIs is configured, we need to look out for changes
+        # in last_ip field as well
```

Comment:
Instead of what, try to write why. If the why is self explanatory, don't add the comment